tensorflow/g3doc/get_started/os_setup.md

```diff
@@ -30,7 +30,7 @@ images are listed in the corresponding installation sections.
 If you encounter installation errors, see
 [common problems](#common_install_problems) for some solutions.
```

Comment:
There are still a couple of these left... can you look for #[a-z_]* and fix them all so we can tick off the files you've touched?